mandatory/2-function-creation.js

```diff
@@ -25,8 +38,41 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+// function tidyUpStrings(arrayOfStrings) {
+//   return.trim().toLowerCase().replace(/\//g, "");
+// }
 
+// const tidyUpStrings =[
+//   "/Daniel",
+//   " /Sanyia",
+//   "AnTHonY",
+//   "irina",
+//   " Gordon",
+//   "ashleigh   ",
+//   "   Alastair  ",
+//   " anne marie  ",
+// ];  
+// function tidyUpStrings(arrayOfStrings) {
+//   return (arrayOfStrings.trim().toLowerCase().replace(/\//g, ""));
+// }
+
+  // const arrayOfStrings = [
+  //   "/Daniel",
+  //   " /Sanyia",
+  //   "AnTHonY",
+  //   "irina",
+  //   " Gordon",
+  //   "ashleigh   ",
+  //   "   Alastair  ",
+  //   " anne marie  ",
+  // ];
+  //const newStrings = arrayOfStrings.map(tidyUpStrings);
+
+function tidyUpStrings(arrayOfStrings) {
+  console.log(arrayOfStrings);
```

Comment:
It looks like you've left a console.log here while you were testing. I would suggest either commenting this line out or removing it entirely.

Reply:
> It looks like you've left a console.log here while you were testing. I would suggest either commenting this line out or removing it entirely.

I have corrected this. Thank you.